scripts/irods/test/test_native_rule_engine_plugin.py

```diff
@@ -92,6 +92,9 @@ def assert_presence_of_attr_value_avu(attr, value, assert_true=True):
             assert_presence_of_attr_value_avu(attr, value, assert_true)
 
     def helper_test_pep(self, rules_to_add, icommand, strings_to_check_for=['THIS IS AN OUT VARIABLE'], number_of_strings_to_look_for=1):
+        # Restart the server here because sometimes the log just stops receiving messages for some unknown reason.
```

Comment:
Is there an issue for this?

Reply:
I don't think so. I'm not even sure that this affects 4.3 since the logging system is different from 4.2 where this is cherry-picked from. Maybe we should try without the change to see if it's needed...?

Reply:
This may be resolved by the move to syslog, but let's confirm that first.

If we can't confirm it, then let's remove the call to restart and keep it moving. We can always open more issues.

Reply:
Tests seem to pass without these, so, I guess we don't need them.